### ISSUE TITLE
sQTL analysis

### ISSUE THREAD

**Issue by isthisthat** [davidaknowles/leafcutter#15]
Hello, there is no mention of how sQTL analysis is done (which is claimed in the paper). I have completed steps 1 & 2 to get clustered junctions for my dataset and I also have genotypes for these samples (vcf). Is there a way to get the variants associated with the creation of novel splice junctions or the loss of a junction? Thank you very much.

**Comment by goldenflaw** [comment]
We are working on making the scripts available. I would suggest you follow the procedure as described in the LeafCutter preprint (http://biorxiv.org/content/early/2016/03/16/044107) to qqnorm+standardize your intron ratio data, and use fastQTL(http://fastqtl.sourceforge.net/) for QTL mapping.

**Comment by goldenflaw** [comment]
I'll try to make those scripts available right after US Thanksgiving. In
general, we would not remove associations where a SNP overlaps junction
reads if WASP was used to re-map reads, but we did not re-align every bam
file. In order to do a proper analysis, you must use WASP or similar
methods to remove all allelic reads that might map incorrectly or not at
all. This is time consuming and one of the reasons why I did not initially
release code for sQTL mapping: it would produce false positives if WASP was
not used.

If I recall correctly, sQTLseekeR tries to find novel isoforms, but
estimates isoform level changes, which is known to be noisy. It finds many
fewer sQTLs compared to most other methods.

Yang
